naga/src/back/hlsl/help.rs

```diff
@@ -380,7 +380,7 @@ impl<W: Write> super::Writer<'_, W> {
         module: &crate::Module,
         constructor: WrappedConstructor,
     ) -> BackendResult {
-        let name = crate::TypeInner::hlsl_type_id(constructor.ty, module.to_ctx(), &self.names)?;
+        let name = TypeInner::hlsl_type_id(constructor.ty, module.to_ctx(), &self.names)?;
```

Comment:
Can we please keep changes like this out of the PR? It makes the change much harder to review, for me at least.